net.krafting.SemantiK.Lang.English.yml

```diff
@@ -0,0 +1,32 @@
+app-id: net.krafting.SemantiK.Lang.English
+build-extension: true
+separate-locales: false
+branch: '1.0.0'
```

Comment:
Do language packs needs to be branched ? This is used in case there are ABI breaks.

I think you can omit `version` in the extension point and omit this here

Reply:
they would likely not cause ABI breaks, I was unsure if this was a required thing.

I remove it from here and a PR is waiting in the main App https://github.com/flathub/net.krafting.SemantiK/pull/6